openshift/init_tenant.go

```diff
@@ -145,6 +144,7 @@ func do(config Config, callback Callback, username, usertoken string, templateVa
 // or default to the OOTB template included
 func loadTemplate(config Config, name string) ([]byte, error) {
 	teamVersion := config.TeamVersion
+	logCallback := config.LogCallback
```

Comment:
'init' will kinda work, but without that part it won't e.g. update resources that has the same name. Things that are auto setup by the ProjectRequest on os but need changes for us to use it. (some of these things are related to specific oso things, but still..) That function is the _main_ logic of the whole thing. Everything else is primarily plumming.

https://github.com/fabric8io/fabric8-init-tenant/blob/master/openshift/apply.go#L231